cocotb/monitors/xgmii.py

```diff
@@ -53,6 +53,10 @@ class XGMII(Monitor):
     Assumes a single vector, either 4 or 8 bytes plus control bit for each byte.
 
     If interleaved is ``True`` then the control bits are adjacent to the bytes.
+
+    .. versionchanged:: 1.4.0
```

Comment:
Spelling out the ".0" hasn't been done so far IIRC, but may be good practice actually...

Reply:
Could go either way. The reality is we shouldn't be changing things in a way that needs documenting in bugfix releases anyway.

Perhaps worth a sweep through all of these for consistency before release.